src/main/java/com/dev/cinema/dao/AbstractDao.java

```diff
@@ -1,19 +1,24 @@
 package com.dev.cinema.dao;
 
 import com.dev.cinema.exceptions.DataProcessingException;
-import com.dev.cinema.util.HibernateUtil;
 import org.apache.log4j.Logger;
 import org.hibernate.Session;
+import org.hibernate.SessionFactory;
 import org.hibernate.Transaction;
 
 public class AbstractDao<T> {
     private static final Logger logger = Logger.getLogger(AbstractDao.class);
+    public final SessionFactory sessionFactory;
```

Comment:
can we have this field protected ? 
if yes, it's recommended to have access modifier as strict as possible

Reply:
Agree thank you